com.vagueentertainment.Framed.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.vagueentertainment.Framed</id>
+  <name>Framed</name>
+  <summary>Digital Picture frame for your NFTshowroom art and personal work</summary>
+  <description>
+    <p> Early version of Framed
+    </p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
```

Comment:
Is it intended that there's no source code available on GitHub?

Reply:
I have the source code in a private repo at the moment, and plan to set it public when its a little closer to 1.0. Though I'm not sure if Godot games can be built via command line so having the binary version might be the only way to get it into the flatpak... I'll do a bit more research.

Reply:
There are some games like https://github.com/flathub/com.github.dariasteam.cowsrevenge so I believe it's possible. Just curious, not a blocker.

Reply:
Good to know. Also thanks for all the help. I'm sure I'll be less of a pest next time around.